### ISSUE TITLE
flake8(erred) on Python 3.7 due to warning?

### ISSUE THREAD

**Issue by jaraco** [SublimeLinter/SublimeLinter-flake8#91]
I have flake8 installed in Python 3.7.0b3. When I run Flake8 on a package with no lint, I see this output:

```
/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/pycodestyle.py:113: FutureWarning: Possible nested set at position 1
  EXTRANEOUS_WHITESPACE_REGEX = re.compile(r'[[({] | []}),;:]')
```

Even though the return value is 0.

The warning is ignorable and probably will be fixed by subsequent releases of flake8 and pycodestyle. In the meantime, I suspect it's what's causing the `flake8(erred)` indication in ST3.

![image](https://user-images.githubusercontent.com/308610/39436224-65d73fbe-4c6b-11e8-973b-0365e8bc7730.png)

If I uninstall flake8 on Python 3.7 and install it for Python 3.6, the `(erred)` goes away.

Should these warnings be suppressed or at least not indicate that the linter failed?

**Comment by kaste** [comment]
You're using beta stuff, you're out. Lol.

On the pro: it should still work even though it says erred. It should still report and highlight the actual lint errors if any. And erred would indicate that there is some output additionally that you should read/react to.

Ah well. I don't know if FutureWarning is something we should expect and suppress. It would be easy to catch though. Is this something flake8 will issue in the future. Will flake8 have an arg to suppress these warnings?

**Comment by jaraco** [comment]
I suspect flake8 will not and probably should not suppress the warnings, and now that I think about the interface that this plugin has with the linter, it probably would have to whitelist lines by pattern to ignore them. So probably the best thing to do is just wait for the new release of flake8 for those who wish to use Python 3.7.

**Comment by h-jia** [comment]
I figured this error by adding follow codes in SublimeLinter.sublime-settings --User
{	
	"linters": {
	"flake8": {
	"@disable": false,
	"args": [],
	"builtins": "",
	"excludes": [],
	"executable": ["/Users/yourname/miniconda3/bin/python", "-m", "flake8"],
	"ignore": "E501",
	"jobs": "1",
	"max-complexity": -1,
	"max-line-length": null,
	"select": "",
	"show-code": false,
},
}
}
